airbyte-protocol/models/src/main/java/io/airbyte/protocol/models/CatalogHelpers.java

```diff
@@ -157,12 +157,4 @@ public static JsonNode fieldsToJsonSchema(List<Field> fields) {
     return allFieldNames;
   }
 
-  public static Set<String> getStreamNames(ConfiguredAirbyteCatalog catalog) {
```

Comment:
oops sorry this is me when I formatted the files. I was going to get rid of it in a follow up PR so up to you to leave this here or remove it.

it's not used any where in our code today.

Reply:
Its fine. Lets keep it